src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateMethodTests.cs

```diff
@@ -7246,7 +7246,7 @@ public void M1()
 parseOptions: TestOptions.Regular);
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/15508"), Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
```

Comment:
Agree. This is an ask mode change to RC.  We can't introduce new IDE regressions here without explicit IDE buy off.  Otherwise this will not pass the ask mode review.